python_modules/libraries/dagster-dg-cli/dagster_dg_cli/cli/api/API_IMPLEMENTATION_PLAN.md

```diff
@@ -0,0 +1,413 @@
+dg api user create --email <email> --role <role>
+dg api user update <user-id> --role <role>
+dg api user delete <user-id>
+```
+
+**GraphQL**: `usersOrError`
+
+### 7. **team**
+
+```bash
+dg api team list [--json]
+dg api team view <team-id> [--json]
+dg api team create --name <name> [--members <user-id1,user-id2>]
+dg api team update <team-id> [--add-member <user-id>] [--remove-member <user-id>]
+dg api team delete <team-id>
+```
+
+**GraphQL**: `teamPermissions`
+
+### 8. **alert-policy**
+
+```bash
+dg api alert-policy list [--json]
+dg api alert-policy view <policy-id> [--json]
+dg api alert-policy create --name <name> --config <file>
+dg api alert-policy update <policy-id> --config <file>
+dg api alert-policy delete <policy-id>
+```
+
+**GraphQL**: `alertPolicies`, `alertPolicyById`
+
+### 9. **code-location**
+
+```bash
+dg api code-location list [--json]
+dg api code-location view <location-name> [--json]
+dg api code-location create --name <name> --image <image> --config <file>
+dg api code-location update <location-name> --image <image> [--config <file>]
+dg api code-location delete <location-name>
+```
+
+**GraphQL**: `repositoriesOrError`, `repositoryOrError`
+
+### 10. **workspace**
+
+```bash
+dg api workspace list [--json]
+dg api workspace view [--json]
+dg api workspace update --config <file>
+# Workspace is typically singular per deployment
+```
+
+**GraphQL**: `workspaceOrError`
+
+---
+
+## **Tier 3: Extended Management**
+
+### 11. **check**
+
+```bash
+dg api check list [--asset <asset-key>] [--status <status>] [--json]
+dg api check view <check-name> --asset <asset-key> [--json]
+# Checks are defined in code, not directly manageable
+```
+
+**GraphQL**: `assetCheckExecutions`
+
+### 12. **schedule**
+
+```bash
+dg api schedule list [--location <name>] [--status <running|stopped>] [--json]
+dg api schedule view <schedule-name> --location <name> [--json]
+dg api schedule start <schedule-name> --location <name>
+dg api schedule stop <schedule-name> --location <name>
+```
+
+**GraphQL**: `schedulesOrError`, `scheduleOrError`
+
+### 13. **sensor**
+
+```bash
+dg api sensor list [--location <name>] [--status <running|stopped>] [--json]
+dg api sensor view <sensor-name> --location <name> [--json]
+dg api sensor start <sensor-name> --location <name>
+dg api sensor stop <sensor-name> --location <name>
+```
+
+**GraphQL**: `sensorsOrError`, `sensorOrError`
+
+### 14. **backfill**
+
+```bash
+dg api backfill list [--status <status>] [--json]
+dg api backfill view <backfill-id> [--json]
+dg api backfill create --asset <asset-key> --partitions <range>
+dg api backfill cancel <backfill-id>
+```
+
+**GraphQL**: `partitionBackfillsOrError`, `partitionBackfillOrError`
+
+### 15. **env-var**
+
+```bash
+dg api env-var list [--location <name>] [--json]
+dg api env-var view <var-name> [--location <name>] [--json]
+# Environment variables managed through deployment settings
+```
+
+**GraphQL**: `utilizedEnvVarsOrError`
```

Comment:
confusingly this is not Dagster+ secrets/environment variables (environment variables also are not managed through deployment settings as i understand that term)

utilizedEnvVarsOrError seems to be something like which env vars are required by resources in a dagster repository or something

Reply:
just going to eliminate it for now